server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -50,9 +50,11 @@ import (
 	inpb "github.com/buildbuddy-io/buildbuddy/proto/invocation"
 	pgpb "github.com/buildbuddy-io/buildbuddy/proto/pagination"
 	pepb "github.com/buildbuddy-io/buildbuddy/proto/publish_build_event"
+	sipb "github.com/buildbuddy-io/buildbuddy/proto/stored_invocation"
 	uidpb "github.com/buildbuddy-io/buildbuddy/proto/user_id"
 	api_common "github.com/buildbuddy-io/buildbuddy/server/api/common"
 	api_config "github.com/buildbuddy-io/buildbuddy/server/api/config"
+	olapdb_config "github.com/buildbuddy-io/buildbuddy/server/olapdb/config"
```

Comment:
instead of aliasing, why don't we call the package `olapdbconfig`? 
i'm not a big fan of the proliferation of `config` packages.

Reply:
Done